openapi/src/test/java/io/helidon/openapi/TestUtil.java

```diff
@@ -293,6 +309,18 @@ public static HttpURLConnection getURLConnection(
         return conn;
     }
 
+    static HttpURLConnection getURLConnection(
+            int port,
+            String method,
+            String path,
+            String queryParameter) throws Exception {
+        URL url = new URL("http://localhost:" + port + path + "?" + queryParameter);
+        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+        conn.setRequestMethod(method);
+        System.out.println("Connectiong: " + method + " " + url);
```

Comment:
Please remove the system out. Also do we need to use HttpURLConnection?

Reply:
I have removed the `System.out`. 

The new `HttpURLConnection` code follows the same pattern as the code that was already in the test; I used the same approach in the new test so it would be very similar to what is already there for ease of understanding by readers.

I know we have more recent approaches for creating client connections in new tests. I avoided rewriting the test as part of this PR to avoid changes not related to the Open API spec version update.